src/utils/validation.js

```diff
@@ -0,0 +1,18 @@
+const validator = require("validator");
+const validateSignUpData = (req) => {
+  const { firstName, lastName, email, password } = req.body;
+
+  if (!firstName && !lastName) {
+    throw new Error("Enter a valid name");
```

Comment:
`lastName` is not required in the schema but is a required field in the validator